src/constants/links.js

```diff
@@ -1,14 +1,15 @@
 export const header = {
-  navDropDowns: [ // dropdowns
+  navDropDowns: [
+    // dropdowns
     // first dropdown start
     [
       {
         LABEL: 'navigation.dropdowns.0.linkLabels.0',
-        LINK: 'link-one',
+        LINK: 'home',
```

Comment:
`/home`?

Reply:
for some reason `home` and `/` both work but `/home` doesn't

Reply:
I completely forgot about the language code in the pathname; this is fine how it is. I'm not sure if Reach Router can help us or if it's beyond the scope of Reach Router? 

Can having `<Router basepath={i18n.language} />` help us in this case?

Reply:
I'll look into it!

Reply:
I think if we wanted to use a base path we could use [withPrefex](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#add-the-path-prefix-to-paths-using-withprefix) from gatsby

Reply:
We could use `withPrefix` if the prefix were static. However, our prefix is dynamic. If you take a look [here](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/path-prefix/), the prefix is done in `gatsby-config`, where i18n is not available.